pkg/injector/test/test_helpers.go

```diff
@@ -0,0 +1,102 @@
+package test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+	"gopkg.in/yaml.v2"
+
+	"github.com/openservicemesh/osm/pkg/logger"
+)
+
+// All the YAML files listed above are in this sub-directory
+const directoryForYAMLFiles = "test_fixtures"
```

Comment:
can we move the roundtrip files to a directory on their own (without any code)?

Reply:
Moved these out to `./tests/envoy_xds_expectations/`